test/arcade/potts/env/location/PottsLocationTest.java

```diff
@@ -829,9 +828,9 @@ public void splitVoxels_YZPlaneDirectionRandomZero_updatesLists() {
         voxelsBSplit.add(new Voxel(1, 1, 1));
         voxelsBSplit.add(new Voxel(2, 2, 2));
         voxelsBSplit.add(new Voxel(2, 2, 0));
-
+        
         Voxel center = loc.getCenter();
-        Plane yzPlane = new Plane(center, Direction.YZ_PLANE.getVector());
+        Plane yzPlane = new Plane(center, Direction.YZ_PLANE.vector);
```

Comment:
```suggestion
        Plane yzPlane = new Plane(center, Direction.YZ_PLANE);
```

You can use the new constructor for all these calls